packages/core/src/resolveTrustChains/resolveTrustChains.ts

```diff
@@ -59,24 +65,92 @@ export const resolveTrustChains = async (options: Options): Promise<Array<TrustC
 
     if (entityStatementChain.some((statement) => statement.exp < now)) {
       // Skip expired chains
+      // TODO: Think about how we want to share this conclusion with the caller
       continue
     }
 
-    // TODO: Merge all the policies and check them against the metadata of the leaf entity
+    if (entityStatementChain.length === 1) {
+      // When there is only one statement, we can assume that the leaf is also the trust anchor
+      const leafEntityConfiguration = entityConfigurationChain[0]
+      if (!leafEntityConfiguration)
+        throw new OpenIdFederationError(PolicyErrorStage.Validation, 'No leaf entity configuration found')
```

Comment:
we have this code below as well. I think we can place it above this if statement, and then do the if?